docs/dashboards/data-pool-and-connectors.md

```diff
@@ -117,6 +117,36 @@ async function run() {
 run();
 ```
 
+## GoogleSheetsConnector Tutorial
+
+Google Sheets is a spreadsheet application included as part of the free, web-based Google Docs Editors suite offered by Google.
+
+With Highcharts Dashboards' `GoogleSheetsConnector` is it possible to load data from a Google spreadsheet and insert it into the connector's data table, allowing the data to be rendered in any dashboard component, whether in a DataGrid, KPI, or chart. Like with other connectors,
```

Comment:
I feel like this is too technical of what is happening under the hood. It should be simpler like...
> With the `GoogleSheetsConnector` in Highcharts Dashboards you can load tables from a Google spreadsheet right into one your dashboard components, whether it is a DataGrid, KPI, or chart.